<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>171</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="nplogin">
    <item row="1" column="1">
     <widget class="QLineEdit" name="pwl">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pw">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nnl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>nickname</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nn">
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>nnl</zorder>
   <zorder>pw</zorder>
   <zorder>pwl</zorder>
   <zorder>nn</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>账号密码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>验证码</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二维码</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>LoginWindow</receiver>
   <slot>loginwindow_save_password()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LoginWindow</receiver>
   <slot>loginwindow_loginfunc_combox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>loginwindow_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>get_sms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginwindow_save_password()</slot>
  <slot>loginwindow_loginfunc_combox()</slot>
  <slot>loginwindow_login()</slot>
  <slot>get_sms()</slot>
 </slots>
</ui>
